vtkext/private/module/vtkF3DImguiObserver.cxx

```diff
@@ -30,7 +30,7 @@ namespace
       { "End", ImGuiKey_End },
       { "Insert", ImGuiKey_Insert },
       { "Delete", ImGuiKey_Delete },
-      { "Backspace", ImGuiKey_Backspace },
+      { "BackSpace", ImGuiKey_Backspace },
```

Comment:
Is this coverable ?

Reply:
Actually on macOS, the symkey is `Backspace` is a lowercase `s` while on Windows it's `BackSpace` is a uppercase `S`.  
I will change the code to translate both to `ImGuiKey_Backspace`.  
The backspace key is covered in `TestInteractionConsoleTypingSceneInfo`.

Reply:
Sounds like a VTK bug I missed during my interaction rework in VTK. Ill fix it, but you still need these two anyway.

Reply:
https://gitlab.kitware.com/vtk/vtk/-/merge_requests/11738